agr_api/src/main/java/org/alliancegenome/api/rest/interfaces/AlleleRESTInterface.java

```diff
@@ -43,6 +46,21 @@ public interface AlleleRESTInterface {
 		@APIResponse(responseCode = "200", description = "Search for alleles.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = Null.class))) })
 	Allele getAllele(@Parameter(in = ParameterIn.PATH, name = "id", description = "Search for an Allele by ID", required = true, schema = @Schema(type = SchemaType.STRING)) @PathParam("id") String id);
 
+	@GET
+	@Path("/{id}/constructs")
+	@Operation(description = "Searches for transgenic alleles", summary = "Transgenic Alleles")
+	@APIResponses(value = { @APIResponse(responseCode = "404", description = "Missing transgenic alleles", content = @Content(mediaType = "text/plain")),
+		@APIResponse(responseCode = "200", description = "Search for alleles.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = Null.class))) })
+	JsonResultResponse<TransgenicAlleleDocument> getAlleleConstructs(@Parameter(in = ParameterIn.PATH, name = "id", description = "Search for an constructs of a given Allele by ID", required = true, schema = @Schema(type = SchemaType.STRING)) @PathParam("id") String id);
+
+	@GET
+	@Path("/{alleleID}/construct/{constructID}")
```

Comment:
@cmpich are these endpoint replacing other endpoints? or are they suppose to be new?